.gitignore

```diff
@@ -70,4 +64,7 @@ yarn-error.log
 # Yarn Integrity file
 .yarn-integrity
 
-.now
\ No newline at end of file
+.now
+
+# Editor files
+.idea/
```

Comment:
We also want to ignore IDE files from git. I use Jetbrains which places its own `.idea` directory in the folder.